modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/DatePicker/DatePicker.es.js

```diff
@@ -154,7 +152,7 @@ const DatePicker = ({
 			});
 			maskInstance.current.update(inputRef.current.value);
 		}
-	}, [inputMask, dateMask, inputRef]);
+	}, [inputMask, dateMask]);
```

Comment:
@alinedoleron, why did you remove the `inputRef` from the `useEffect` array of dependencies? Won't it generate any side effect?

Reply:
Hi @natocesarrego, I removed because inputRef doesn't change. But I can put it back, anyway.